lib/is-under-scope.js

```diff
@@ -133,11 +160,13 @@ var stripPseudoSelectorsFromScopeList = function(scopeList) {
 // Another way to think about it: Can the target scope cascade properties to the node?
 //
 // For scope-lists see: `generateScopeList`
-var isUnderScope = function(nodeScopeList, scopeNodeScopeList, /*optional*/ignorePseudo) {
-	// Because we only care about the scopeNodeScope matching to the nodeScope
-	// Remove the pseudo selectors from the nodeScope so it can match a broader version
-	// ex. `.foo:hover` can resolve variables from `.foo`
-	nodeScopeList = stripPseudoSelectorsFromScopeList(nodeScopeList);
+var isUnderScope = function(nodeScopeList, scopeNodeScopeList, /*optional*/ignorePseudo, /*optional*/keepPseudo) {
```

Comment:
Can we document this dichotomy between `ignorePseudo` and `keepPseudo`? I'm not quite wrapping my head around why we need both options and them being opposite default states. Seems like they should both be `ignoreX` or `keepX` and maybe need better descriptors.

For example, just from an initial look, seems like `ignorePseudo` can cover `keepPseudo` by being `ignorePseudo=false`. But I understand they are probably tackling different use cases. So the parameter name should probably be different.